modules/gapi/src/backends/cpu/gcpustereo.cpp

```diff
@@ -0,0 +1,48 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#include <opencv2/gapi/stereo.hpp>
+#include <opencv2/gapi/cpu/stereo.hpp>
+#include <opencv2/gapi/cpu/gcpukernel.hpp>
+
+#include <opencv2/calib3d.hpp>
+
+GAPI_OCV_KERNEL_ST(GCPUStereo, cv::gapi::calib3d::GStereo, cv::gapi::calib3d::StereoSetup)
```

Comment:
Please see my answers in discussion about `cv::gapi::calib3d::StereoSetup` as a state.

Reply:
Done I hope...

Reply:
Can you please explain why it is stateful?

Reply:
Its stateful for not to create  `cv::gapi::calib3d::StereoSetup` every time kernel is called

Reply:
Also its needed for keep approved kernel API and also have opportunity to configure `blockSize`, `numDisparity` and etc.